### PR TITLE
apt-file: add Indonesian translation

### PR DESCRIPTION
**Pull request by punkestu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, can you translate the placeholders (i.e. ones within `{{}}` that don't have these options like `search|find`, `show|list` and `search|find`)?

**Review by kbdharun — APPROVED** [review]

File: pages.id/linux/apt-file.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# apt-file
+
+> Mencari file-file di dalam paket `apt`, termasuk yang belum diinstal.
```
Comment (reinhart1010) [inline pages.id/linux/apt-file.md:3]:
```suggestion
> Cari kumpulan berkas di dalam paket `apt`, termasuk yang belum dipasang.
```

Can you please change this (and subsequent) sentences to use imperative mood (see [this](https://share.reinhart1010.id/tldrguidelinesidspecific) and [this](https://share.reinhart1010.id/tldridimperativemood))?

Additionally, as a maintainer I currently intend to deprecate `file` in favor of `berkas` for Indonesian translations.

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for you contribution. Here are some of my reviews.

File: pages.id/linux/apt-file.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# apt-file
+
+> Cari kumpulan berkas di dalam paket `apt`, termasuk yang belum dipasang.
+> Informasi lebih lanjut: <https://manpages.debian.org/latest/apt-file/apt-file.1.html>.
+
+- Memperbarui basis data metadata:
```
Comment (reinhart1010) [inline pages.id/linux/apt-file.md:6]:
```suggestion
- Perbarui basis data metadata:
```

File: pages.id/linux/apt-file.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`sudo apt update`
+
+- Mencari paket yang berisi nama file atau path yang diinputkan:
```
Comment (reinhart1010) [inline pages.id/linux/apt-file.md:10]:
```suggestion
- Cari paket yang berisi nama file atau path yang diinputkan:
```

File: pages.id/linux/apt-file.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`apt-file {{search|find}} {{path/ke/sebuah/file}}`
+
+- Menampilkan daftar konten dari sebuah paket:
```
Comment (reinhart1010) [inline pages.id/linux/apt-file.md:14]:
```suggestion
- Tampilkan daftar konten dari sebuah paket:
```

File: pages.id/linux/apt-file.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`apt-file {{show|list}} {{paket}}`
+
+- Mencari paket yang yang sesuai dengan `ekspresi_reguler`:
```
Comment (reinhart1010) [inline pages.id/linux/apt-file.md:18]:
```suggestion
- Cari paket yang sesuai dengan `ekspresi_reguler`:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Hi, can you also apply these changes? Thank you!

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Pull request merged**